test/command/latex-stackengine.md

```diff
@@ -0,0 +1,18 @@
+# stackengine
```

Comment:
This should be renamed `test/command/6898.md` -- that makes it easier for us to locate the issue to which a test corresponds.